crates/runtime/src/http/v1/assist.rs

```diff
@@ -240,8 +241,11 @@ pub(crate) async fn post(
     )
     .label("tables".to_string(), format!("{input_tables:?}"));
 
+    let tspan = tracing::span!(target: "task_history", tracing::Level::INFO, "vector_search", input_text = %payload.text);
+
     let relevant_data = match vs
         .search(payload.text.clone(), input_tables, RetrievalLimit::TopN(3))
+        .instrument(tspan)
```

Comment:
We already trace practically the same span (both with name `"vector_search"`), in `vs.search`. 
https://github.com/spiceai/spiceai/pull/2257/files#diff-2099bcaabb894d8accf6f48632574ffa7633d2c2b9c4d56383a84b7e1e8c7a4fR195